io.kopia.KopiaUI.json

```diff
@@ -0,0 +1,63 @@
+{
+    "app-id": "io.kopia.KopiaUI",
+    "base": "org.electronjs.Electron2.BaseApp",
+    "base-version": "19.08",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "19.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "kopia-ui",
+    "separate-locales": false,
+    "rename-desktop-file": "kopia-ui.desktop",
+    "rename-icon": "kopia-ui",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=pulseaudio",
+        "--share=network",
+        "--device=dri",
+        "--filesystem=host",
```

Comment:
What does it need host access for?

Reply:
It is a backup program. In order to backup the whole system, it needs access to the whole system.